docs/content/about-iota/iota-wallet/how-to/restore-account.mdx

```diff
@@ -0,0 +1,50 @@
+---
+description: A guide on how to restore an account using a seed or a .stronghold file.
+teams:
+   - iotaledger/tooling
+tags: [iota-wallet]
+---
+# Restore From Seed or Stronghold
+
+If you have lost your mnemonic, you can still recover your account provided you have access to your seed or
+`.stronghold` file.
+
+## Back Up a Seed
+
+You can export you seed by clicking on the `...` menu, and then clicking `Backup Seed`.
+
+   ![Backup Seed](/img/about-iota/iota-wallet/how-to/restore-account/backup-seed-falafel.png)
+
+Firefly will then prompt you to enter your password.
+
+   ![Enter Password](/img/about-iota/iota-wallet/how-to/restore-account/enter-password.png)
+
+   ![Copy seed](/img/about-iota/iota-wallet/how-to/restore-account/copy-seed.png)
+
+With your seed at hand, you can [import your wallet using the IOTA Web Wallet](../getting-started.mdx#importing-an-existing-wallet).
+
+## Get a Seed From a `.stronghold` file
+
+You can use your `.stronghold` backup file to obtain your wallet's seed.
+
+If don't know where your `.stronghold` file is, click on the `...` menu and select the `Copy system profile location`. 
```

Comment:
this step doesnt really make much sense here imo, if you are in firefly already you can simply export the seed within firefly without the need to locate the file and then import it. This is just a tip we can provide users as fyi but with firefly open they should use the backup seed feature that already comes with the selected profile

Reply:
We could leave it in, but idk seems weird imo

Reply:
Added an admonition and changed the title to make it clear this are two different options